client/console_conf.go

```diff
@@ -0,0 +1,33 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package client
+
+// ConsoleConfStartResponse is the response for starting the console-conf
+// routine.
+type ConsoleConfStartResponse struct {
+	ActiveSnapAutoRefreshChanges []string `json:"active-snap-refreshes,omitempty"`
```

Comment:
There's a discrepancy between the JSON name and the Go name. Please decide if both or neither should use the word `auto`.

Reply:
ah good catch thanks! yes both should have auto in them I think